js/main.js

```diff
@@ -19,9 +20,57 @@ $entryForm.addEventListener('submit', function (event) {
   data.entries.unshift(formInputValue);
   $photoPreview.src = './images/placeholder-image-square.jpg';
   $entryForm.reset();
+  var list = renderEntryDomTree(formInputValue);
+  entryHolder.prepend(list);
+  entryHolder.scrollIntoView();
```

Comment:
Do you have both views on the page at the same time? One should be hidden while the other is shown. You will want to make use of the `hidden` class from `layout.css`